modules/animationqt/src/animationeditordockwidgetqt.cpp

```diff
@@ -268,11 +299,69 @@ AnimationEditorDockWidgetQt::AnimationEditorDockWidgetQt(
     }
 
     toolBar->addSeparator();
+
     controller_.AnimationControllerObservable::addObserver(this);
 }
 
 AnimationEditorDockWidgetQt::~AnimationEditorDockWidgetQt() = default;
 
+void AnimationEditorDockWidgetQt::openAnimation() {
+    InviwoFileDialog openFileDialog(this, "Import Animation ...", "animation");
+    openFileDialog.addSidebarPath(PathType::Workspaces);
+    openFileDialog.addExtension("inv", "Inviwo File");
+    openFileDialog.setFileMode(FileMode::AnyFile);
+
+    if (openFileDialog.exec()) {
+        QString path = openFileDialog.selectedFiles().at(0);
+        std::string fileName{utilqt::fromQString(path)};
+        fileName = filesystem::cleanupPath(fileName);
+        if (!filesystem::fileExists(fileName)) {
+            LogError("Could not find file: " << fileName);
+            return;
+        }
+        try {
+            std::ifstream anim{std::string(fileName)};
```

Comment:
use `filesystem::ifstream()` in order to handle unicode paths.

https://github.com/inviwo/inviwo/blob/20222bd6170a488ce8cf18df32f2fff0931c25b9/include/inviwo/core/util/filesystem.h#L107-L108

Reply:
why `std::string(fileName)` ?